### DISCUSSION TITLE
declarationProcessingStatus BEVFELD_CHECK mit jelenet?

### DISCUSSION THREAD

**Discussion by szalaierika** [nav-gov-hu/eVAT#45] [Kérdések és válaszok - Gépi interfész]
Kedves NAV Szakértők,

A queryDeclarationProcessingStatusResponse-ban a declarationProcessingStatus BEVFELD_CHECK mit jelenet pontosan?

Köszönjük előre is!

**Comment by renced42** [comment]
Kedves @szalaierika 
Ez azt jelenti, hogy a beküldést követő kibontás és feldolgozás követően az elkészített bevallás XML-t a NAV bevallásfeldolgozó rendszerében ellenőrzi az eÁFA  rendszer és ad adott pillanatban ebben a státuszban van a bevallás.
